pkg/minishift/hostfolder/sshfs_hostfolder.go

```diff
@@ -75,7 +75,7 @@ func (h *SSHFSHostFolder) Mount(driver drivers.Driver) error {
 	// Mount command seems to fail occasionally. Give it a couple of attempts
 	mount := func() (err error) {
 		cmd := fmt.Sprintf(
-			"sudo sshfs docker@%s:%s %s -o IdentityFile=%s -o 'StrictHostKeyChecking=no' -o reconnect -o allow_other -o idmap=none -p %d",
+			"sudo sshfs docker@%s:%s %s -o compression=no -o kernel_cache -o IdentityFile=%s -o 'StrictHostKeyChecking=no' -o reconnect -o allow_other -p %d",
```

Comment:
the default value for `idmap` is none, so removed it.
```
 -o idmap=TYPE          user/group ID mapping, possible types are:
             none             no translation of the ID space (default)
             user             only translate UID of connecting user
             file             translate UIDs/GIDs contained in uidfile/gidfile
```

Reply:
have you verified this to be the case on several distros?

Reply:
Also, are these options essential and/or in what way to they provide benefit?

Reply:
@gbraad It is like this for the b2d and centos iso, these are not essential, but the issue was to use some _additonal_ options which would improve performance, now i haven't done a benchmark on this but these are supposed to improve the performance, as no compression could speed up the process and kernel_cache would use the cache..

Reply:
`compression=no` might speed up on older machines, but it does consume less IOPS for the actual transfer. although this might even be possible to be offloaded on some systems. so, yeah... I am not sure it actually improves without evidence.

other options we have added before, but their effect were instantly noticable... as for instance it enabled SELinux, etc. Here, I am not really convinced

Reply:
@gbraad I will try to find some benchmarks if there is any for optimizing sshfs.

Reply:
Especially as what is changed here is not in line with the title of the commit. It feels like additional options are added (which are perhaps even configurable).

Note: pre-mature optimization is never a good idea, especially without identifiable improvements. this would be a change without a clear reason. what would `kernel_cache` do? it makes no sense without more details or even an explanation.

Reply:
@gbraad From the help section: `kernel_cache` caches files in kernel. normal caching is by-default turned on, this allows the kernel to also cache, this will speed up subsequent reads.

The other option `large_files` is also for speeding up reads. by reading large chunks we don't have to make frequent read operations, so less `read()` and `open()` calls, i assume.